zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/FastLeaderElection.java

```diff
@@ -1114,6 +1138,74 @@ public Vote lookForLeader() throws InterruptedException {
         }
     }
 
+    private Vote followingBehavior(Map<Long, Vote> recvset, Map<Long, Vote> outofelection, SyncedLearnerTracker voteSet, Notification n) {
```

Comment:
i like how you factored this out, but i don't like the name. (i'm terrible with naming.) perhaps followerVoteCalculation?